intermediate_source/speech_command_recognition_with_torchaudio.py

```diff
@@ -0,0 +1,523 @@
+"""
+Speech Command Recognition with torchaudio
+==========================================
+
+This tutorial will show you how to correctly format an audio dataset and
+then train/test an audio classifier network on the dataset.
+
+Colab has GPU option available. In the menu tabs, select “Runtime” then
+“Change runtime type”. In the pop-up that follows, you can choose GPU.
+After the change, your runtime should automatically restart (which means
+information from executed cells disappear).
+
+First, let’s import the common torch packages such as
+``torchaudio <https://github.com/pytorch/audio>``\ \_ that can be
```

Comment:
Looks like the link format should be markdown, not RST format.

https://colab.research.google.com/notebooks/markdown_guide.ipynb#scrollTo=70pYkR9LiOV0

Reply:
Thanks for pointing out that the conversion from notebook to rst didn't go well for the link :) Since the tutorial is first rendered on the pytorch website, rst is used, e.g. [there](https://github.com/pytorch/tutorials/blob/master/intermediate_source/char_rnn_classification_tutorial.py#L60).

Reply:
Indeed, the conversion from colab to rst didn't render correctly here. Thanks for pointing it out! The tutorial is rendered first on pytorch.org, so I see other tutorials using rst links, e.g. [here](https://github.com/pytorch/tutorials/blob/master/intermediate_source/char_rnn_classification_tutorial.py#L60).